core/Http.php

```diff
@@ -534,6 +534,10 @@ public static function sendHttpRequestBy(
                     $status = (int)$m[2];
                 }
 
+                foreach ($http_response_header as $line) {
```

Comment:
Cheers for this fix @stevenseeley 

wondering... do we maybe need to move it down a few lines so it is also used when using `fopen` or would `http_response_header` not be set in that case?

Reply:
Actually yes, outside of the if statement would probably be more ideal. I’ll test and update on Monday this pull.